sympy/core/tests/test_relational.py

```diff
@@ -17,7 +17,7 @@ def test_rel_ne():
 
     # issue 6116
     p = Symbol('p', positive=True)
-    assert Ne(p, 0) is S.true
```

Comment:
These tests should be kept. We still want that behavior.

Reply:
For the current master:
```
>>> from sympy import *
>>> from sympy.core.relational import Ne
>>> p=symbols('p')
>>> assert Ne(p, 0) is S.true
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
>>> 

```
But, I don't know how its clearing the tests. Can you please check it once.

Reply:
p must be positive.

```
from sympy import Ne, Symbol, S
p = Symbol('p', positive=True)
```
gives:
```
In [8]: Ne(p, 0) == S.true
Out[8]: True

In [9]: Ne(p, 0) is S.true
Out[9]: True
```

The thing is that if you change the tests you must be sure that the new behavior is the behavior you want.